spec/amber/cli/commands/exec_spec.cr

```diff
@@ -43,13 +43,18 @@ module Amber::CLI
     end
 
     context "outside of project" do
-      it "complains if not in the root of a project" do
-        expected_result = "Error: 'amber exec' can only be used from the root of a valid amber project"
+      it "executes outside of project but without including project" do
```

Comment:
Perhaps this is not easy to do, but how is this test verifying that a project is not getting included?

Reply:
The the next test verifies that part. This test just verifies that it still runs.